### PR TITLE
Closes #1839 Dummy function created

### PR DESCRIPTION
**Pull request by ashachakma**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiraldev/devel/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/devel/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/cran-release/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Comment by bms63** [comment]
Make sure to go through each checklist item and complete when relevant - if not relevant then just sign off as well.

Also - make a section in the News for your update

File: R/my_first_fcn.R
Comment (bms63) [inline R/my_first_fcn.R:1]: This needs a roxygen header 
image

File: R/test-my_first_fcn.R
Comment (bms63) [inline R/test-my_first_fcn.R:1]: this should be removed

Comment (ashachakma) [reply R/test-my_first_fcn.R:1]: File removed and new changes committed

File: R/my_first_fcn.R
Comment (ashachakma) [reply R/my_first_fcn.R:1]: Header updated

File: renv.lock
Comment (bms63) [inline renv.lock:1]: this file shouldn't be updated

**Review by bms63 — COMMENTED** [review]
do a quick update to the news file as well.

**Comment by bms63** [comment]
Hi @ashachakma just to wrap this up:

* can you click the update branch button to get that experience
image
* Create a blurb in the `news.md` file

Then we can close this out.

**Comment by ashachakma** [comment]
> Hi @ashachakma just to wrap this up:
> 
> * can you click the update branch button to get that experience
>   image
> * Create a blurb in the `news.md` file
> 
> Then we can close this out.

Its done now.

**Comment by bms63** [comment]
oh - actually see if you can fix the spelling check that is failing.  that is a good one to know.

**Pull request closed without merging**